db/seeds.rb

```diff
@@ -1,7 +1,53 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+    require 'faker'
```

Comment:
I cloned down the repo, dropped the database and tried to seed it. 

I got: ActiveRecord::RecordInvalid: Validation failed: Name is too short (minimum is 5 characters)